scripts/README.md

```diff
@@ -291,3 +291,27 @@ function build_release() {
 
 main $@
 ```
+### Using release.sh to publish to Azure
+To run release.sh to publish to Azure Contianer Registery and Azure Blob, the following pre-requisites need to be configured/setup:
+#### A. Install Tools
+1. Install `az` command line tool, see [here] (https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+2. Install `azcopy` tool, see [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10)
+
+#### B. Create ACR
+1. Create an Azure Container Registry.
+2. Get the and save the admin key locally, see [here](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-authentication).
+
+#### C. Create Azure BLOB
+1. Create an Azure Storage account
+2. Create a "Container" in the blob, instructions [here](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-portal).
+3. Assign read/write permissions to the user/Service Principle that will be used, see [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-auth). If you see 404s when you run the release script, you likely havent set permissions correctly.
+
+#### Configure tests
+Automated azure support for running integration tests (ie AKS setup etc.) has not tested. The `--test-args` flag can be used to restrict presubmit testing to unit and build tests only.
```

Comment:
Or you can skip tests altogether with --skip-tests, not sure how useful just running the build/unit tests are at this point.

Reply:
tests are tests - the more the merrier so would like to run as many as possible :)

Reply:
That's fair. But using --skip-tests is useful when you actually don't care about tests but do care about getting a build quickly. That's better developer experience. It's worth documenting --skip-tests here since you had the trouble to add --test-args.